src/__mocks__/View.js

```diff
@@ -9,7 +9,9 @@
 
 'use strict';
 
-var createReactNativeComponentClass = require('createReactNativeComponentClass');
+var createReactNativeComponentClass = require(
+  'createReactNativeComponentClass',
+);
```

Comment:
The nice thing is that prettier always seem to put `from` and the module name of an ES import on the same line so at least you can grep `from 'x'` easily.